src/main/resources/default/taglib/k/link.html.pasta

```diff
@@ -1,5 +1,5 @@
 <i:arg type="String" name="link" description="Defines a url or link."/>
-<i:arg type="String" name="name" description="Defines a speaking name or description."/>
+<i:arg type="String" name="name" default="" description="Defines a speaking name or description."/>
```

Comment:
should we then also skip the &nbsp below? or, we could add s.th. like `<i:if test="isFilled(name)"><span class="ml-2">@name</span></i:if>` - maybe ml-1 - need t ocheck this. I think we do this elsewhere to get consistent spacing between text and icons.

Reply:
thats a good call. I am going to add this

Reply:
ml-1 should be sufficient

Reply:
done